tests/resolvers/Mutation/removeSampleOrganization.spec.ts

```diff
@@ -69,57 +69,24 @@ afterAll(async () => {
 });
 
 describe("Remove Sample Organization Resolver - User Authorization", async () => {
-  it.skip("should NOT throw error when user is ADMIN", async () => {
```

Comment:
And previously this test use to get skipped as it was resulting in error. So I modified the test to perform the expected functionality.